Lib/test/test_bisect.py

```diff
@@ -186,6 +186,11 @@ def test_optionalSlicing(self):
                         self.assertTrue(data[ip-1] <= elem)
                     self.assertEqual(ip, max(lo, min(hi, expected)))
 
+    def test_hi_backcompatbility(self):
+        for func, data, elem, expected in self.precomputedCases:
+            self.assertEqual(func(data, elem, 0, -1),
```

Comment:
This doesn't check that passing `-1` and `None` is the same as not specifying the hi argument. Compare results with `func(data, elem)`.